content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,55 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers from users to controllers:
+
+First the user signals to the controller that it should take ownership of a
+field, usually by setting some other field that the controller observes. The
+controller then claims ownership of the field by applying a configuration
+containing the field. This can be done in one of the following ways:
+
+- If the controller is designed to immediately set the value of the field, the
+controller should apply with force set to true. Once the user observes that the
+controller has taken ownership of the field, they can remove the field from
+their applied configuration to complete the transfer.
+- If the controller will not set the value of the field immediately, the user
```

Comment:
This is the ordinary case right now, since no controller specifically uses SSA yet, and therefore it cannot take ownership of a field until it actually writes the object in a way that changes the field. 

So maybe this should go first?

Reply:
Yeah, let's lead with that.

Reply:
Fixed. I've also qualified the other case a bit more to make it clear that the user must be able to wait to observe the controller take ownership of the field.